tests/replacement.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Perhaps just include this in the existing `reconfiguration.py` test?

Reply:
Yes, you're right.

Reply:
So that turns out to be awkward, because the test leaves the network with f dead nodes. But then that's probably a bad thing in the suite sooner or later too, so I think it's best to address that.